### ISSUE TITLE
JAWS 18 is not reading aria-labelledby  and aria-describedby elements contents when attribute is added to role="dialog"

### ISSUE THREAD

**Issue by lifter035** [FreedomScientific/VFO-standards-support#3]
### Summary

> JAWS 18 is not reading aria-labelledby  and aria-describedby element contents when attribute is added to role="dialog"
> 1. Go to https://codepen.io/lifter035/project/full/AxKWap/
> 2. "Open dialog" button will be focused but JAWS will not announce even that (another issue?)
> 3. Activate the button to open dialog
> 4. Dialog will autofocus to first radio button in the group

### Expected result

> JAWS should announce dialog aria-labelledby and aria-describedby elements contents when dialog is shown (display:none; style changed to display:block;)

### Actual result

> JAWS is reading only focused radio button label
> JAWS is reading only close button label if dialog opened via "Open dialog - focus on close" button
> Also if I change role="dialog" to role="alertdialog" JAWS is announcing dialog title while dialog is still "display:none;"

#### JAWS version and build number

> Example: JAWS 18.0.4104
 
#### Operating System and version

> Example: Windows 10 Enterprise Evaluation 64-bit

#### Browser and version:

> Example: Internet Explorer 11 (haven't tested others)

**Comment by stevefaulkner** [comment]
Thanks @lifter035 I have been able to reproduce the bug in JAWS18+IE11. Having also tested with JAWS 16/17+IE11and found that I could not reproduce with those versions, it looks like it is a regression. I will file an internal bug on this and report back.

**Comment by lifter035** [comment]
Thanks @stevefaulkner

**Comment by stevefaulkner** [comment]
@lifter035 note: in your codepen the `aria-describedby="decription"` has an incorrect value, decription/description

**Comment by lifter035** [comment]
@stevefaulkner fixed now. It's still the same. One thing I noticed is that if I shift-tab out of the dialog, and then tab back to close button, then JAWS reads title and description now.
But if I tab out of dialog from Radio buttons and then Shift-TAB back to it which focuses current radio button, it doesn't read title or description

**Comment by stevefaulkner** [comment]
@lifter035 yes it does not change the result, the current JAWS behavior is somewhat funky and so will file a bug. Also there is another dialog issue https://github.com/FreedomScientific/VFO-standards-support/issues/6

**Comment by stevefaulkner** [comment]
@lifter035  Internal bug filed - thanks!
Bug #96906 - accessible name and description not announced for role=dialog

**Comment by AariasPapa** [comment]
Is this issue related to JAWS18 in general or is it a Specific version of JAWS18 that has this issue. I am having the same issue where its not reading the aria-labledby and ria-describedby for role=dialog and role=alertdialog and trying to pin point the issue.

**Comment by ggordon-vfo** [comment]
This works for me with JAWS 2019 in IE, Chrome, and Firefox. I'm closing the issue.
